packages/ketcher-core/src/application/editor/Editor.ts

```diff
@@ -43,6 +45,12 @@ export class CoreEditor {
     this.drawingEntitiesManager = new DrawingEntitiesManager();
     this.domEventSetup();
     this.canvasOffset = this.canvas.getBoundingClientRect();
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    editor = this;
+  }
+
+  static provideEditorInstance(): CoreEditor {
+    return editor;
```

Comment:
Yes it can be. However editor instance is creating in react package after canvas initialization. So when user starts the interaction with editor it should be created already.